Description: Don't use newline in error message

At least intake forwards error messages by a mechanism that
can't handle them, and hence fails tests

(Upstream fixed this in intake instead:
https://github.com/intake/intake/pull/699
but we're in transition freeze)

Author: Rebecca N. Palmer <rebecca_palmer@zoho.com>
Forwarded: https://github.com/dask/dask/issues/9813

--- a/dask/backends.py
+++ b/dask/backends.py
@@ -137,7 +137,7 @@
                 except Exception as e:
                     raise type(e)(
                         f"An error occurred while calling the {funcname(func)} "
-                        f"method registered to the {self.backend} backend.\n"
+                        f"method registered to the {self.backend} backend.  "
                         f"Original Message: {e}"
                     ) from e
 
